common/src/Assets/AttributeDefinition.h

```diff
@@ -53,6 +53,14 @@ namespace TrenchBroom {
             Type type() const;
             const String& shortDescription() const;
             const String& longDescription() const;
+            /**
+             * Returns a description of the options for ChoiceAttributeOption and FlagsAttributeDefinition,
+             * other subclasses return an empty string.
+             */
+            virtual String optionDescriptions() const;
```

Comment:
This method only applies to one particular subclass, it should not be in the interface. I would make it private and rename it something more general, like `additionalDescription` or something along those lines.

Reply:
Wait, I now see that it is called from the UI code in order to format the text etc, so I guess it has to be here. But I wonder if we cannot remove the fullDescription method. I don't think the full description should be displayed in a tooltip anyway.

Reply:
Yeah, we can remove it. Tooltips can use shortDescription(), and nothing else calls it.